llvm/lib/Support/Unix/Signals.inc

```diff
@@ -413,10 +413,22 @@ static void SignalHandler(int Sig, siginfo_t *Info, void *) {
     raise(Sig);
 #endif
 
-  // Signal sent from another process, do not assume that continuing the
-  // execution would re-raise it.
-  if (Info->si_pid != getpid())
+  // Signal sent from another userspace process, do not assume that continuing
+  // the execution would re-raise it.
+  if (Info->si_pid != getpid() && Info->si_pid != 0) {
+#if defined(__linux__) || defined(__ANDROID__)
```

Comment:
No need to check `__ANDROID__` here, Android also defines `__linux__`.

Reply:
Looks like `__ANDROID__` is still there.

Reply:
I could have sworn I removed that... should be gone now.